ospf6d/ospf6_message.c

```diff
@@ -2356,7 +2356,20 @@ static void ospf6_send_lsupdate(struct ospf6_neighbor *on,
 	}
 	if (oi) {
 		ospf6_packet_add(oi, op);
-		OSPF6_MESSAGE_WRITE_ON(oi);
+		/* If ospf instance is being deleted, send the packet
+		 * immediately */
```

Comment:
Please fix this warning:

Report for ospf6_message.c | 2 issues
===============================================
< WARNING: Block comments use a trailing */ on a separate line
< #2360: FILE: /tmp/f1-27009/ospf6_message.c:2360: